dashboard.el

```diff
@@ -61,12 +61,10 @@
   :syntax-table nil
   :abbrev-table nil
   (buffer-disable-undo)
-  (whitespace-mode -1)
-  (linum-mode -1)
-  (when (>= emacs-major-version 26)
-    (display-line-numbers-mode -1))
-  (when (require 'page-break-lines nil t)
-    (page-break-lines-mode 1))
+  (when (featurep 'whitespace) (whitespace-mode -1))
+  (when (featurep 'linum) (linum-mode -1))
+  (when (featurep 'display-line-numbers) (display-line-numbers-mode -1))
+  (when (featurep 'page-break-lines) (page-break-lines-mode 1))
```

Comment:
Even calling `(minor-mode -1)` will load it's module. Avoid doing that can increase startup time.